src/views/PrivacyView.vue

```diff
@@ -17,10 +17,10 @@ const navigateToHome = () => {
     </div>
     <div class="col-12">
       <Card>
-        <template #title> Datenschutzerklärung </template>
+        <template #title> {{ $t('privacy.title') }}</template>
         <template #content>
-          <p>Im Folgenden möchten wir Sie aufklären, wie Ihre Daten von uns verarbeitet werden.</p>
-          <p class="p-card-content-header">Verantwortlich im Sinne der DSGVO ist:</p>
+          <p>{{ $t('privacy.explanation') }}</p>
+          <p class="p-card-content-header">{{ $t('privacy.responsibility') }}</p>
           <p>
             Prof. Dr.-Ing. Alexander Stanik<br />
```

Comment:
It’s not necessary to translate the text here, but it would be better to manage and maintain all text consistently in the JSON file for easier updates and uniformity.